src/components/main/flight/FlightInfoScreen.js

```diff
@@ -239,265 +377,357 @@ export default class FlightInfoScreen extends React.Component {
-          </View>
+        <View
+          style={{
+            flex: 1,
+            flexDirection: "column",
+            alignSelf: "center",
+            justifyContent: "center",
+          }}>
+          <Text
+            style={{
+              color: COLORS.forestgreen,
+              textAlign: "center",
+              fontFamily: "Poppins-bold",
+              fontSize: Math.round(moderateScale(30, 0.0625)),
+            }}>
+            Insert Loading Screen Here :)
+          </Text>
         </View>
-      )
-    }
-    else {
+      );
+    } else {
       return (
-        <View style={styles.container}>
-          <View style={styles.containerTop}>
-            <View style={styles.buttonBarNav}>
-              {/*Navigation Buttons*/}
-              <Feather style={styles.navigationIcon} name="home"
-                onPress={() => this.props.navigation.navigate('Home')} />
-              <FontAwesome style={styles.navigationIcon} name="user-circle-o"
-                onPress={() => this.handleUserRedirect()} />
-            </View>
-            <View style={styles.buttonBarTop}>
-              {/*Route Option Buttons*/}
-              <View style={[styles.leftGreenButton, { opacity: (this.state.tripIndex == 1) ? 1 : 0.5 }]}>
-                <TouchableOpacity
-                  onPress={() => this.setState({ tripIndex: 1 })}>
-                  <Text style={styles.buttonText}>{i18n.t('ONE WAY')}</Text>
-                </TouchableOpacity>
-              </View>
-              <View style={[styles.rightGreenButton, { opacity: (this.state.tripIndex == 2) ? 1 : 0.5 }]}>
-                <TouchableOpacity
-                  onPress={() => this.setState({ tripIndex: 2 })}>
-                  <Text style={styles.buttonText}>{i18n.t('ROUND TRIP')}</Text>
-                </TouchableOpacity>
+        <ImageBackground
+          source={this.background}
+          style={styles.background_image}>
+          <SafeAreaView style={styles.backDrop}>
+            <View style={styles.innerView}>
+              {/* Flight Information */}
+              <View style={styles.topInnerView}>
+                <View style={styles.flightView}>
+                  <Text style={styles.flightNumberLabel}>Flight Number</Text>
+                  <Text style={styles.flightNumberText}>
+                    {flightChars}
+                    {flightNums}
+                  </Text>
+                  <Text style={styles.flightInfoText}>
+                    {depCityName} ({depCityIata}) to {arrCityName} (
+                    {arrCityIata}) via {airlineName}{" "}
+                  </Text>
+                </View>
+
+                {/* Arrival Departure */}
+                <View style={styles.itineraryView}>
+                  <View style={styles.itineraryViewItem}>
+                    <Text style={styles.itineraryLabel}>{depAirportName}</Text>
+                    <Text style={styles.itineraryAirportCode}>
```

Comment:
This is a result of wrapping. Even if they were two different rows. This would cause the second to be a little bit further down if the name is too long. IATA codes are constant sizes, 3 letters. However, Airport names are a slippery slope. If you have independent rows, then Column 1 will have tons of unnecessary white space, and column 3 will be filled. Row 2 will be 1 line but...eh just creates another problem.

Reply:
Sure go ahead with that. We'll need to remember to ask Flavio's input on that afterwards though

Reply:
Tbh, in my opinion Flavio should be reviewing the results of our PR's once we are all done. If he has any questions on why X was taken out it is explained to him by you or me or Jules because he does not code....The designer basically has had no interface with the dev's either, which I understand given the circumstances. But if the designer knew what was gud they would have done this more Developer friendly XD.

Reply:
Nah, QA should be reviewing that it works (except we don't have a QA team) Reviews come from pre-release builds once we get a significant portion done.

Also, any UI element is doable, (not easy per se).

Reply:
^^^ evidently, but we can't stop text wrapping unless we parse smarter.

Reply:
The text wrapping is fine. As long as the double lines and the single lines are centered vertically. Did you check out the fix I commented btw? I out it in the wrong thread (the one below this)